charts/dashboards/cluster.json

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
Do you think it's worth adding info to the dashboard docs about the new dashboards?

Reply:
I think the observability topic probably deserves a whole section (metrics + logging)

Reply:
Ok, we can make a separate ticket for this, and do it once the pod metrics are in too